main.py

```diff
@@ -8,69 +8,98 @@
 import os
 import os.path
 from aiohttp import web
+import json
 
-
-# Customize the cookie name, value and cookie expire date. 
+# Customize the cookie name, value and cookie expire date.
 # The cookie will be stored 30 days long.
 COOKIE_NAME = "OldTamil"
 COOKIE_VALUE = 0
-COOKIE_LIFE_TIME = 24 * 60 * 60 * 30   
-
-
-# User credentials.
-USERNAME = 'Hippod'
-PASSWORD = 'admin'
+COOKIE_LIFE_TIME = 24 * 60 * 60 * 30
 
 # Define the required host and port.
 HOST = '127.0.0.1'
 PORT = 8888
 
 
+# Check for the requested file in the system, return server error! if file not found.
+def check_file(filename):
+    if not os.path.isfile(filename):
+        print("Internal server error! file not found: {}".format(filename))
+        return False
+    response = open(filename).read()
+    return response
+
```

Comment:
naming is unclear here: check_file should check the file, not more. name your functions after their purpose!